app/code/Magento/Sales/Model/Order.php

```diff
@@ -1326,7 +1326,7 @@ public function getTrackingNumbers()
      */
     public function getShippingMethod($asObject = false)
     {
-        $shippingMethod = parent::getShippingMethod();
```

Comment:
Clarification why it was changed: static test was complaining that parent method `getShippingMethod` doesn't exists. I checked - it's really doesn't exists, so instead it goes to magic method `__call` and runs `getData` method

Reply:
Just fixed and all back to normal

Reply:
That was actually fix for static tests, that I've commited to your branch :)

Could you revert your commit "roll back changes", squash all changes and force push?

Reply:
Ok my bad. it's now reverted. i was thinking that i messed up the code during changes, without realising that you commit it more changes to make the tests pass.
Sry and thanks !